project-base/docker/conf/docker-compose-win.yml.dist

```diff
@@ -0,0 +1,86 @@
+version: "3.1"
+services:
+    postgres:
+        image: postgres:9.5-alpine
+        container_name: shopsys-framework-postgres
+        volumes:
+            - pgdata:/var/lib/postgresql/data
+        environment:
+            - PGDATA=/var/lib/postgresql/data
+            - POSTGRES_USER=root
+            - POSTGRES_PASSWORD=root
+            - POSTGRES_DB=shopsys
+
+    webserver:
+        image: nginx:1.13-alpine
+        container_name: shopsys-framework-webserver
+        working_dir: /var/www/shopsys-framework
+        volumes:
+            - .:/var/www/shopsys-framework
+            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
+        ports:
+            - "8000:8080"
+        links:
+            - php-fpm
+            - smtp-server
+
+    php-fpm:
+        build: project-base/docker/php-fpm
```

Comment:
Does it work without the build args? If I'm not mistaken, there are no default values for the argument in the Dockerfile.